From 5569bdc7dfeaf3974255d635982717102dd3e766 Mon Sep 17 00:00:00 2001
From: "Munoz, Obed N" <obed.n.munoz@intel.com>
Date: Tue, 29 Sep 2015 12:43:57 -0500
Subject: [PATCH] Add python3 support with open function

Signed-off-by: Munoz, Obed N <obed.n.munoz@intel.com>
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index ff70b70..b1d6063 100644
--- a/setup.py
+++ b/setup.py
@@ -33,7 +33,7 @@ version_string = "1.2.2"
 
 description = "Kerberos high-level interface"
 
-long_description = file(joinpath(dirname(__file__), "README.rst")).read()
+long_description = open(joinpath(dirname(__file__), "README.rst")).read()
 
 url = "http://www.calendarserver.org/"
 
-- 
2.6.0

